docs/nodes/providers/2_setting_up.md

```diff
@@ -20,17 +20,6 @@ Check latest version [here](https://github.com/JackalLabs/canine-provider/releas
 
 :::
 
-Install make and confirm installation. 
```

Comment:
@Tieman0220 are you 100% sure? I installed go and was forced to install make still. This has happened to me twice now on both my provider installations.

Reply:
I have only ever had to just install Go and it's worked for me every time so far. Granted most if the time I do this I don't create a separate user, but the one time I did make a jackal user I don't remember having to do this either.

Reply:
Perhaps leave this section in because for the casual user:

a) If it turns out it's already installed, trying to install again won't hurt them
b) if they do in fact need to install it, the error message they get from trying to build without installing might confuse them a great deal

We both have conflicting evidence so best to leave it in and err on the side of caution?

Reply:
resolved w/ latest commit